### DISCUSSION TITLE
Auto-format citations in publication (CSL) style

### DISCUSSION THREAD

**Discussion by Dr-QTDS** [aidenlx/obsidian-zotero#511] [Ideas]
When inserting references using this plugin, is it possible to design the reference format? For example, in the main text, the citation is annotated as a superscript, and the citation details are shown in the reference part.

使用这个插件在markdown中引入参考文献时，是否可以像写论文一样，指定引用格式，正文部分用上标表示，引文具体内容在参考文献部分？

**Comment by aidenlx** [comment]
implemented in v2.1.0 beta
